apiserver/localofferauth.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017 Canonical Ltd.
```

Comment:
I've discussed verbally previously to validate the approach used in this PR but do intend to follow up again now that this has work been done.